types/deno/.eslintrc.json

```diff
@@ -0,0 +1,10 @@
+{
+    "rules": {
+        "@typescript-eslint/adjacent-overload-signatures": "off",
+        "@typescript-eslint/no-empty-interface": "off",
```

Comment:
These kinds of rules pretty much always indicate a problem and we generally prefer them to be disabled at a line-level so we can' more closely see the rationale; do know what all is failing here?

Reply:
I added comments here explaining these ignored lint rules. Basically the types are code generated here (https://github.com/denoland/deno/pull/25545) and it's a bit of work to do that. Additionally, the original types get run through `deno lint`. I made a few fixes in the Deno repo based on the feedback from this tool though.